File: dev/sbt-plugin/src/sbt-test/sbt-plugin-commands/run-all-javadsl/a/impl/src/main/java/impl/FooServiceImpl.java
package impl;

import akka.NotUsed;
import com.lightbend.lagom.javadsl.api.ServiceCall;
import java.util.concurrent.CompletableFuture;
import javax.inject.Inject;
import api.FooService;

import akka.stream.javadsl.Source;

public class FooServiceImpl implements FooService {

  @Override
  public ServiceCall<NotUsed, String> foo() {
    return request -> CompletableFuture.completedFuture("ack foo");
  }
}

File: dev/sbt-plugin/src/sbt-test/sbt-plugin/run-all-scaladsl/a/impl/src/main/scala/impl/FooServiceImpl.scala
/*
 * Copyright (C) 2016-2020 Lightbend Inc. <https://www.lightbend.com>
 */

package impl

import akka.NotUsed
import com.lightbend.lagom.scaladsl.api.ServiceCall
import scala.concurrent.Future
import api.FooService

class FooServiceImpl extends FooService {
  override def foo = ServiceCall { _ =>
    Future.successful("ack foo")
  }
}
